news/2025/2025-09-17-osumania-4k-world-cup-2025-quarterfinals-recap.md

```diff
@@ -0,0 +1,153 @@
+| 9 | Argentina ::{ flag=AR }:: | ::{ flag=ES }:: Spain | RC3 | 1,439 | ::{ flag=AR }:: | ::{ flag=ES }:: | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/119196396 "Match history") |
+| 10 | Hong Kong ::{ flag=HK }:: | ::{ flag=GB }:: United Kingdom | RC4 | 1,605 | ::{ flag=HK }:: | ::{ flag=HK }:: | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/119206077 "Match history") |
+
+At first glance you may notice that the match between ::{ flag=TH }:: Thailand and ::{ flag=JP }:: Japan had not one, not two, but three of the top 10 closest maps throughout the whole round. While going off of the final result it may not look like it, that final scoreline could have been very different.
+
+Thailand picking their protect of [Ariabl'eyeS - Romanesque Romancia](https://osu.ppy.sh/beatmapsets/2428522) (RC3) as the first map of the match ended up being a point for Japan. An unfortunate miss about halfway through the map gave Japan the lead, and from there, they just about managed to hold on even though the accuracies favoured the Thai roster. Right after that map, the Japanese protect of [Puru - Toki (maidable Remix)](https://osu.ppy.sh/beatmapsets/1306576) (RC7) ended up going to Thailand with a single miss on both sides. This resulted in a close point right up until the very end.
+
+And that wasn't even the closest map during the weekend! That map would come right after these 2 break points. On [Azumi Takahashi feat. Lotus Juice - It's Going Down Now](https://osu.ppy.sh/beatmapsets/2428514) (RC1) the Japanese players managed to all keep their combos, and after another unfortunate miss happening on the Thai side, it looked like Japan would run away with yet another break point. But as the map came to a close, and with the accuracy lead on their side, Thailand managed to take the point by the skin of their teeth.
+
+To round off the closest maps of this weekend we have scores from the matches between ::{ flag=PL }:: Poland and the ::{ flag=PH }:: Philippines as well as between ::{ flag=HK }:: Hong Kong and the ::{ flag=GB }:: United Kingdom. Both of these matches had some incredibly close calls that deserve a watch. If you haven't read the highlights up above for these matches, be sure to go back and give them a read. We promise that it will be worth your time!
+
+With Quarterfinals seeing the smallest score differences on average of every weekend thus far, it's exciting to think what we'll see going into the Semifinals. Will we finally see score differences enter the double digits? Or will we perhaps see matches get so close that they have to be decided through the tiebreaker? With players performing as well as they have, and stakes getting increasingly higher, the extraordinary performances are set to continue this weekend as well.
```

Comment:
outdated since semis are over

```suggestion
Whew. We don't know about you, but with such close victories and comebacks, we got to experience plenty of hype moments at the edge of our seats. In fact, the Quarterfinals had the smallest score differences on average of every weekend thus far, so it's exciting to think what we'll see going into future rounds.
```